.gitignore

```diff
@@ -2,3 +2,4 @@
 ui/.cache
 ui/dist
 ui/node_modules/
+cmd/costmodel/costmodel
```

Comment:
?

Reply:
git really wants me to check in the binary